packages/integrations/src/awslambda.ts

```diff
@@ -0,0 +1,230 @@
+import * as Sentry from '@sentry/browser';
```

Comment:
Integrations are standalone "additions" to the SDKs. They have no link to the SDK using it, other than `getCurrentHub` passed in the `setupOnce` method. This import shouldnt be used.

Reply:
made this change in the new pull request.